Documentation/Columns/Properties/DisplayCond.rst

```diff
@@ -207,3 +207,5 @@ Flex form fields can access field values from various different sources:
     <displayCond>FIELD:flexField_1:!=:foo</displayCond>
     <!-- Hide field if value of field "flexField_1" from sheet "sheet_1" is not "foo" -->
     <displayCond>FIELD:sheet_1.flexField_1:!=:foo</displayCond>
+    <!-- Hide field for new records -->
+    <displayCond>REC:NEW:true</displayCond>
```

Comment:
This is not restricted to FlexForms and especially does not fit the purpose of this doc section. This section explains the ways to access field values given the special nature of FlexForms.

Instead this must be moved to the general examples above, then with regular PHP/TCA syntax. Also a 2nd example for `REC:NEW:false` should be added for clarity.

Reply:
Good point

Reply:
Can we do that in a follow up As the current example is not wrong?

Reply:
The example itself is wrong but it is wrongly placed, thus unexpected and won't be discovered by people. So adding it like this would not add as much value as it could.

Reply:
I just updated the change and moved everything upwards.